### DISCUSSION TITLE
About the scale form mpii json

### DISCUSSION THREAD

**Discussion by carolchenyx** [open-mmlab/mmpose#588] [General]
Dear all,  I would like to know where is the scale come from in the mpii_train.json or how to get it? As for the center, is it the center of the bbox? Thank you.

**Comment by jin-s13** [comment]
The center/scale are officially provided by MPII team.
```
 .scale - person scale w.r.t. 200 px height
 .center- rough human position in the image
```
Please refer to [MPII homepage](http://human-pose.mpi-inf.mpg.de) for more information.

**Comment by carolchenyx** [comment]
Thanks jin-s13 for the quick reply, but I would want to know how it comes from not the definition of it. Some researchers thought the center is the bbox center, however I checked it wrong. How is your thinking of it. One reason I would want to know is that I want to convert my dataset into mpii format.

**Comment by jin-s13** [comment]
I am not very sure what is the rough human position for MPII. But I think using the bbox center is reasonable.

**Comment by innerlee** [comment]
Yeah you may consult authors of MPII

**Comment by carolchenyx** [comment]
Yes, it may be a good idea hahahaha.

lizz ***@***.***> 于2021年4月19日周一 下午5:49写道：
